src/UI/logmodel.h

```diff
@@ -1,22 +1,45 @@
 #pragma once
 
-#include <QAbstractListModel>
-#include <vector>
+#include <QAbstractTableModel>
 
-class LogModel : public QAbstractListModel
+struct LogInfo{
+    QString filePath;
+    QString operationType;
+    DWORD pid;
+    DWORD numberOfBytes;
+    DWORD offset;
+    QString preview;
+    QString fileHandle;
+    QString operationTime;
+    QString resultOfTheOperation;
+};
+
+namespace Columns{
```

Comment:
You can just declare `enum class` instead of using namespace and then use column names like with namespace: `Column::Pid`.

Reply:
I changed all unscoped enums to scoped , I used unscoped because of the implicit conversion to int.
But explicit is more clearer.